frontend/app/tab/tabbar.tsx

```diff
@@ -654,7 +671,7 @@ const TabBar = memo(({ workspace }: TabBarProps) => {
         title: "Add Tab",
     };
     return (
-        <div ref={tabbarWrapperRef} className="tab-bar-wrapper">
+        <div ref={tabbarWrapperRef} className={`tab-bar-wrapper${autoHideTabsBar ? '-auto-hide' : ''}`}>
```

Comment:
Rather than use two separate classes, can you create a common class for the shared attributes and a separate `auto-hide` class for the auto hide attributes? I'd also recommend using `clsx` to assemble the class string.